### PR TITLE
feat(crm_brand): add crm_brand

### PR DESCRIPTION
**Pull request by ygol**
This module allows you to manage branded opportunities. It adds a brand field on the sales team and propagate the value on the opportunities.

### CODE REVIEW

File: crm_brand/models/crm_team.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+from odoo import api, fields, models, _
+
+
+class CrmTeam(models.Model):
+
+    _name = 'crm.team'
+    _inherit = ['crm.team', 'res.brand.mixin']
+
+    brand_id = fields.Many2one(
```
Comment (sbejaoui) [inline crm_brand/models/crm_team.py:12]: please consider that brand field is already added to crm.team in sale_brand module, see https://github.com/OCA/brand/blob/12.0/sale_brand/models/crm_team.py#L10

there is a PR to fast forward it to 13.0 #52

Comment (ygol) [reply crm_brand/models/crm_team.py:12]: oh great. so this PR is not needed and I close it. right?

Comment (sbejaoui) [reply crm_brand/models/crm_team.py:12]: yes If you don't need brand on crm.lead

File: crm_brand/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2021 Yves Goldberg
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Crm Brand',
+    'description': """
+        Add brand to CRM""",
+    'version': '13.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'Yves Goldberg',
```
Comment (MeetKD) [inline crm_brand/__manifest__.py:10]: Add oca as author.

File: crm_brand/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    'website': 'https://www.ygol.com',
```
Comment (MeetKD) [inline crm_brand/__manifest__.py:11]: Website should be as per oca guidelines

File: crm_brand/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    'depends': ['brand', 'crm'
+                ],
+    'data': [
+        'views/crm_lead.xml',
+        'views/crm_team.xml',
+    ],
+    'demo': [
+    ],
```
Comment (MeetKD) [inline crm_brand/__manifest__.py:19]: Please Remove the empty key.

File: crm_brand/models/crm_lead.py [hunk 5]
```diff
@@ -0,0 +1,12 @@
+# Copyright 2021 Yves Goldberg
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models, _
+
+
+class CrmLead(models.Model):
+
+    _name = "crm.lead"
```
Comment (MeetKD) [inline crm_brand/models/crm_lead.py:9]: Is this really needed?

**Review by MeetKD — CHANGES_REQUESTED** [review]

**Pull request closed without merging**